src/app/shared/services/affinity-group.service.ts

```diff
@@ -38,11 +38,11 @@ export class AffinityGroupService extends BaseBackendCachedService<AffinityGroup
 
   public updateForVm(
     vmId: string,
-    affinityGroupId: string
+    affinityGroupIds: string
```

Comment:
We should probably make this an `Array<string>` and use array representation of groups instead of string representation everywhere.